src/metabase/models/action.clj

```diff
@@ -86,6 +90,7 @@
                            :implicit ImplicitAction)]
       (db/execute! {:insert-into model
                     :values [(-> (apply dissoc action-data action-columns)
+                                 (mi/add-entity-id)
```

Comment:
I think it's better if we let properties handle entity rather than leave it in `insert!` function like this.

It's easy to miss this, if we introduce new ways to add a subaction without using this function.

Reply:
Not relevant anymore, entity_ids are gone